index.html

```diff
@@ -0,0 +1,1161 @@
+              </div>
+              <div class="card-body p-0 py-2">
+                <h5 class="card-title">Florida Rep Is First of Two Congressmen to Test Positive for Coronavirus</h5>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <div class="d-flex flex-column">
+              <div class="media pb-2 _arial text-dark">
+                <img src="images/content/bill-de-blasio.jpg" class="mr-3" alt="...">
+                <div class="media-body _fnt15"><a href="#" class="text-decoration-none">
+                  <h6 class="mt-0 font-weight-bold text-body">Fines, Jail and Firings: Coronavirus Quarantine Rules Around the World</h6></a>
+                </div>
+              </div>
+              <div class="media border-top py-2 _arial text-dark">
+                <img src="images/content/jobs-unemployment.jpg" class="mr-3" alt="...">
+                <div class="media-body _fnt15">
+                  <a href="#" class="text-decoration-none"><h6 class="mt-0 font-weight-bold text-body">Colorado Sees 'Unprecedented' 1,600 Percent Surge in Unemployment Applications</h6></a>
+                </div>
+              </div>
+              <div class="media border-top pt-2 _arial text-dark">
+                <img src="images/content/jobs-unemployment.jpg" class="mr-3" alt="...">
+                <div class="media-body _fnt15">
+                  <a href="#" class="text-decoration-none"><h6 class="mt-0 font-weight-bold text-body">Colorado Sees 'Unprecedented' 1,600 Percent Surge in Unemployment Applications</h6></a>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- SLIDESHOWS Sports -->
+    <div class="pb-2 px-0 mx-3">
+      <div class="_barlowCond text-danger _fnt22 font-weight-bold my-2 mx-0">
+        Sports
+      </div>
+      <div class="container-fluid px-0 mx-0">
+        <div class="row px-0">
+          <div class="col-md-4">
+            <div class="card rounded-0 border-0 m-0" >
+              <div class="position-relative p-0">
+                <img src="images/content/donald-trump.jpg" class="card-img-top" alt="donald-trump.jpg">
+                <p class="position-absolute py-1 px-3 text-uppercase bg-light text-danger" style="bottom: -16px;">u.s.</p>
+              </div>
+              <div class="card-body p-0 py-2">
+                <h5 class="card-title">Trump Signs Emergency Bill to Manufacture Medical Supplies for Coronavirus</h5>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <div class="card rounded-0 border-0" >
+              <div class="position-relative p-0">
+                <img src="images/content/mario-diaz-balart.jpg" class="card-img-top" alt="donald-trump.jpg">
+                <p class="pos
```

Comment:
- [ ] Please use semantic HTML for the footer.

- [ ] Check the footer when it's extra small and small. The Newsweek copyright text is overlapping the logo.

Reply:
Used semantics for the footer
Corrected overlapping